pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/ProtoWriterTools.java

```diff
@@ -1052,7 +1072,7 @@ public static int sizeOfString(FieldDefinition field, String value) {
             return 0;
         }
         final int size = sizeOfStringNoTag(field, value);
```

Comment:
I wonder if `field` is actually needed in `sizeOfStringNoTag()` It's only used to check for oneOf() in case of empty strings, but for empty strings the size is zero anyway...

Reply:
Yes, it can be removed. I covered the case of null oneOf() case (that required a check for null in `Utf8Tools#encodedLength`, after that the field could be safely removed.